web-docs/docs/_partial/_sys_req_hardware.mdx

```diff
@@ -0,0 +1,37 @@
+import ScDocusaurusConfig from "@site/ScDocusaurusConfig.js";
+
+<>The {ScDocusaurusConfig.title} has been designed to take advantage of the Ignition server's available hardware
+resources. Multi-threading and asynchronous processing allow the connector to simultaneously handle multiple requests at
+a time. </><br/>
+
+#### Minimum Hardware Requirements
+
+CPU: 2 cores<br/>
+RAM: 4 GB<br/>
+
+:::caution
+
+The minimum hardware requirements are suitable for a small number of devices and tags. If you are planning to connect
+many devices or tags, you should consider increasing the hardware resources available to Ignition and its modules.
+
+These requirements match those of Ignition, with additional memory (RAM) required for the connector to run.
+
+:::
+
+#### Recommended Hardware Requirements
+
+CPU: 4 cores<br/>
+RAM: 8 GB<br/>
+
+:::tip
+
+The recommended hardware requirements are suitable for most installations. If you are planning to connect a large number
+devices and/or tags, you should consider increasing the hardware resources available to Ignition and its modules.
+
+There is no upper limit on the number of devices and tags that can be connected to the connector, but the hardware
+resources available to Ignition will limit the number of devices and tags that can be connected. If you experience
+a degradation in performance, it is recommended to increase the hardware resources available to Ignition and its
```

Comment:
Should you mention network as a source of performance degradation?

Reply:
Added a small message that network bandwidth could be increased.

Reply:
I think this probably should get rephrased. The way it reads right now is if you have any performance issue you should get a better cpu, increase ram, increase network bandwidth, and increase hdd space.

a protentional issue could read
Title
"Rephrase performance degradation section in web-docs/docs/_partial/_sys_req_hardware.mdx"
Msg
"Rephrase section to add some protentional troubleshooting steps to identify and resolve the bottleneck"

Reply:
You might even rephrase to say that decreased or sub optimal performance could be due to one or more of the following: cpu availability/speed, memory, network, disk space, etc.

Reply:
Issue #145 was created